src/MockedData.ts

```diff
@@ -0,0 +1,245 @@
+/**
+ * Italia PagoPA Proxy
```

Comment:
this headline is pretty useless, replace it with an explanation of the purpose of this module (same for all modules)